git/git-2.26.0-gui-fixes.patch

```diff
@@ -0,0 +1,91 @@
+--- a/git-gui/git-gui.sh	2020-03-23 02:50:46.000000000 +0300
```

Comment:
I am still of the rather strong opinion that this should be contributed to the Git GUI project by applying this patch on top of https://github.com/gitgitgadget/git's `git-gui/master`, then opening a PR there and submitting it to the Git mailing list for review via the `/submit` command in the PR.

The reason why I say this is that the MSYS2 maintainer (who could merge this patch here) can hardly be asked to review this patch touching idiosyncratic Tcl/Tk code.

And once it gets merged by @prati0100 (the Git GUI maintainer), it is just a matter of time for it to trickle down into v2.27.0 and then we do not need _any_ change here.

Reply:
@dscho, done. Please, see https://github.com/gitgitgadget/git/pull/612. Can you help me to `/allow` on GitGitGadget?

Reply:
Done.